spec/services/moab_record_service/create_spec.rb

```diff
@@ -57,39 +57,41 @@
 
     it_behaves_like 'calls AuditResultsReporter.report_results'
 
-    context 'db update error' do
-      context 'ActiveRecordError' do
-        before do
-          allow(PreservedObject).to receive(:create!).with(hash_including(druid: druid))
-                                                     .and_raise(ActiveRecord::ActiveRecordError, 'foo')
-        end
+    context 'when ActiveRecordError is raised' do
```

Comment:
These are nearly all in support of a more idiomatic RSpec setup since I was already in here fixing a missing stub.